.github/workflows/check-file-size.yaml

```
@@ -0,0 +1,20 @@
+---
+name: Check file size
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  check-file-size:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: read
+    steps:
+    - name: Checkout Branch
+      uses: actions/checkout@v4
+    - name: "Check for oversized files"
+      uses: freenet-actions/check-file-size@v1
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        max_size: 51000
```

We can now reduce it further after https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22157

```suggestion
        max_size: 5100
```